spotbugs-exclude.xml

```diff
@@ -23,6 +23,15 @@
 -->
 <FindBugsFilter>
     <!-- contains specific exclusions of bug patterns -->
+    <Match>
+        <Or>
+            <Class name="org.tweetwallfx.cache.URLContentCacheBase$1" />
+            <Class name="org.tweetwallfx.cache.URLContentCacheBase$2" />
+            <Class name="org.tweetwallfx.config.ConfigurationTest" />
+            <Class name="org.tweetwallfx.stepengine.api.StepEngine" />
```

Comment:
This entry could be removed if the caught and rethrown RuntimeException is wrapped in a e.g. IllegalStateException

Reply:
The remaining exclusions will be needed as implemented interfaces declare `Exceptions` within the `javafx.concurrent.Task::call` and the catch block for `RuntimeException` within the `StepEngine`